### PR TITLE
Fixed design of College Physics Chapter outlines

### PR DESCRIPTION
**Pull request by PatBoj**
It was from #1686.
image

### CODE REVIEW

File: src/scripts/modules/media/body/content-baked.less [hunk 1]
```diff
@@ -178,8 +185,6 @@ h1.example-title .text {
 .note     { .make-block(note); }
 [data-type="example"],
 .example  { .make-block(example); }
-[data-type="abstract"],
```
Comment (Adrianltd) [inline src/scripts/modules/media/body/content-baked.less]: abstract = chapter outline? does it need to happen globally?
if that code doesn't affect anything else badly, I approve

File: src/scripts/modules/media/body/content-baked.less [hunk 2]
```diff
@@ -178,8 +185,6 @@ h1.example-title .text {
...
-.abstract { .make-block(note); }
```
Comment (PiotrKozlowski) [inline src/scripts/modules/media/body/content-baked.less]: This looks like some global mixin for `.abstract`. Should we remove it or just reset in `.os-chapter-outline`? @helenemccarron

**Comment by helenemccarron** [comment]
The learning objectives and chapter outlines are confusing. Some of the chapter outline (using the class/data-type `abstract`) are generated in the content, some (using the class 'chapter-outline` or `learning-objectives`) are created directly in the content. Visually, there shouldn't be a difference, we tried to unify those in the recipes but there is such a descrepancy between books, it's really hard to catch all of them. I would recommend testing several books (biology, physics, economics,  statistics) this should give us a good idea of the range of markup that needs to be supported. Hopefully, this should be as easy as looking at the first chapter of all those books and confirm the fix proposed by Patryk is appropriate.

**Comment by PatBoj** [comment]
@helenemccarron everything I've checked physics, economics, statistics and biology books and they look fine to me.

**Pull request merged**